documentation/modules/post/windows/gather/credentials/adi_irc.md

```diff
@@ -0,0 +1,169 @@
+## Vulnerable Application
+
+  This post-exploitation module extracts clear text credentials from the Adi IRC Client.
+
+  The Adi IRC Client is avaialble from (https://www.adiirc.com/).
+
+  This module extracts information from the config.ini and networks.ini files in the "AppData\Local\AdiIRC" directory.
+
+  This module extracts server information such as server name, server port, user name, and password.
+
+
+## Verification Steps
+
+1. Start MSF console
+2. Get a Meterpreter session on a Windows system
+3. use post/windows/gather/credentials/adi_irc
+4. Set SESSION 1
+5. enter 'run' to extract credentials from all applications
+
+
+## Options
+### VERBOSE
+
+By default verbose is turned off. When turned on, the module will show information on files
+which aren't extracted and information that is not directly related to the artifact output.
+
+
+### STORE_LOOT
+This option is turned on by default and saves the stolen artifacts/files on the local machine,
+this is required for also extracting credentials from files using regexp, JSON, XML, and SQLite queries.
+
+
+### EXTRACT_DATA
+This option is turned on by default and will perform the data extraction using the predefined
+regular expression. The 'Store loot' options must be turned on in order for this to take work.
+
+## Scenarios
+### Default Output
```

Comment:
Instead of default output here could we get the version of Windows and the version of Halloy application.